exopy/app/headless/autoclose.py

```diff
@@ -0,0 +1,7 @@
+from ...measurement.hooks.base_hooks import BasePostExecutionHook
+
+class AutoClose(BasePostExecutionHook):
+    def run(self, workbench, engine):
+        print('  Closing Exopy...')
+        ui = workbench.get_plugin(u'enaml.workbench.ui')
+        ui.stop_application()
```

Comment:
Missing blank line

Reply:
Why is that even needed since you explicitly prevent the UI from even starting ?

Reply:
It's not needed for this patch to work but this is the first approach we tried and we thought this hook could be useful for someone else.